sugondat-chain/pallets/blobs/src/tests.rs

```diff
@@ -99,8 +102,8 @@ fn test_max_total_blob_size_reached() {
 }
 
 #[test]
-#[should_panic = "Maximum total blob size exceeded"]
-fn test_max_total_blob_size_exceeded() {
+#[should_panic = "Maximum total blobs size exceeded"]
+fn test_max_total_blobs_size_exceeded() {
```

Comment:
oh sorry, got it! I made the change because I called the storage values used to track the size `TotalBlobsSize`, so I've made the mistake twice 👍🏻  I will correct it then! thanks!